### DISCUSSION TITLE
My 2023 podcast listening list - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#33] [General]
# My 2023 podcast listening list - Alan Pope's blog

Podcast consumption At the start of 2021, I wrote up my 2021 podcast listening list. In it, I categorised podcasts I listen to broadly as ‘Must listen’, ‘Regular listens’ and ‘Once in a while’.
Back then I was using PocketCasts on Android. The full list of subscriptions can be found here.
Android ➡️ iOS In June 2022, I switched from Android to iOS as I moved from OnePlus 5 to iPhone 13 Mini as my primary device.

https://popey.com/blog/2023/09/my-2023-podcast-listening-list/

**Comment by keedon** [comment]
You might want to try corecursivve. It's a pretty good technical podcast

**Reply by popey** [reply]
Thanks for the suggestion, I'll give them a listen.
